python/tests/test_storage.py

```diff
@@ -20,7 +20,7 @@
 import seldon_core
 from minio import Minio, error
 import unittest.mock as mock
-from utils import skipif_gcs_missing
+from .utils import skipif_gcs_missing
```

Comment:
Presumably this is a slip and is also the reason linting is failing?

Reply:
So it wasn't that one, it was a different file - https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1103/1.log#L44